source/containers.tex

```diff
@@ -17329,8 +17329,8 @@
       { x.swap(y); }
 
   private:
-    container_type @\exposid{c}@;       // \expos
-    key_compare @\exposid{compare}@;    // \expos
+    container_type @\exposid{c}@;           // \expos
+    key_compare @\exposid{compare}@;        // \expos
```

Comment:
I refuse to learn the editorial "rules" for comments, so someone else needs to sign off on these two (obivously editorial) lines.